app/jobs/searchgov_domain_indexer_job.rb

```diff
@@ -2,9 +2,9 @@ class SearchgovDomainIndexerJob < ActiveJob::Base
   queue_as :searchgov
 
   def perform(searchgov_domain:, delay:)
-    searchgov_domain.searchgov_urls.unfetched.first&.fetch
+    searchgov_domain.searchgov_urls.fetch_required.first&.fetch
```

Comment:
The new `SearchgovUrl.fetch_required` scope includes both new URLs and URLs with a `lastmod` value later than `last_crawled_at`.